app/segments/drill/[id]/assignment/index.js

```diff
@@ -42,12 +45,29 @@ export default function Index() {
 
   const { showDialog, showSnackBar } = useAlertContext();
 
+  const isAssignedToday = (assignedTime) => {
+    const today = new Date();
```

Comment:
if we're rounding all timestamp to the day, then we can just compare all timestamps directly